src/Native/System.Native/pal_errno.cpp

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#define _POSIX_C_SOURCE
```

Comment:
Oops, this is not necessary and leftover from some attempts at getting the posix signature for strerror_r everywhere. In the end, I just used configure to detect which one we have, which is better because the GNU one is presumably more efficient for the fixed, recognized error case.